floors/enforce_test.go

```diff
@@ -1154,7 +1219,66 @@ func TestUpdateEnforcePBS(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			got := updateEnforcePBS(tt.args.enforceFloors, tt.args.reqExt)
 			assert.Equal(t, tt.want, got, tt.name)
+		})
+	}
+}
+
+func TestIsBidPriceMoreThanBidfloor(t *testing.T) {
 
+	tests := []struct {
+		name      string
+		bidFloor  float64
+		bidPrice  float64
+		precision float64
```

Comment:
The `precision` field does not seem to be used.

Reply:
Removed unused variable